README.md

```diff
@@ -64,6 +64,39 @@ in the bundle (perhaps because of out-of-date dependencies).
 
 * `--noroot`: By default, source-map-explorer finds common prefixes between all source files and eliminates them, since they add complexity to the visualization with no real benefit. But if you want to disable this behavior, set the `--noroot` flag.
 
+## API
+### `explore(filePathOrContent[, sourceMapPathOrContent][, options])`
+* `filePathOrContent` <[string]|[Buffer]> - path to file or Buffer with contents
+* `sourceMapPathOrContent` <[string]|[Buffer]> - path to source map or Buffer with contents
+* `options` <[Object]> Options for generation
+  * `onlyMapped` <[boolean]> (default `false`) See above for details
+  * `html` <[boolean]> (default `false`) When true html will be included in returned object
+  * `noRoot` <[boolean]> (default `false`) See above for details
+  * `replace` <[Object]<{ [from: [string]]: [string] }>> Mapping for replacement, see above for details
+* returns: <[Object]>
+  * `totalBytes` <[number]>
+  * `numUnmapped` <[number]>
+  * `counts` <[Object]<{ [sourceFile: [string]]: [number] }>> Map containging files extracted from the source map an how much space (bytes) the take inside of provided file. Additinal key  `<unmapped>` is included if `options.onlyMapped` is `false`.
```

Comment:
I know `counts` is the internal variable name, but as an external variable it doesn't make sense. Maybe `files` would be a better name?